Sources/MoyaSugar/SugarTargetType.swift

```diff
@@ -59,15 +48,12 @@ public extension SugarTargetType {
     return self.route.method
   }
 
-  public var parameters: [String: Any]? {
-    return self.params?.values
+  public var task: Task {
+    return self.defaultTask
```

Comment:
How about just getting rid of `params` and make `task` as a required property? It seems that we don't need `params` anymore.

Reply:
Sounds good.